### PR TITLE
Filter trails

### PR DESCRIPTION
**Pull request by arelra**
## What does this change?

Filter trails

### CODE REVIEW

**Review by mxdvl — APPROVED** [review]

**Comment by mxdvl** [comment]
It shall be noted that I did not review of the code and rely on @ioannakok's expertise, here!

File: dotcom-rendering/src/layouts/LiveLayout.tsx [hunk 1]
```diff
@@ -12,7 +12,10 @@ import {
 } from '@guardian/source-foundations';
 import { Hide } from '@guardian/source-react-components';
 import { StraightLines } from '@guardian/source-react-components-development-kitchen';
-import { palette as darkModePalette } from '../../src/palette';
+import {
+	palette as darkModePalette,
+	palette as themePalette,
+} from '../../src/palette';
```
Comment (mxdvl) [inline dotcom-rendering/src/layouts/LiveLayout.tsx:18]: Funky!

**Review by mxdvl — COMMENTED** [review]
For consistency, I wonder if this could have been done as a method that can be passed to `Array.filter` like the `decideTrail` function is passed to `Array.map`. Composition is nice!

**Comment by arelra** [comment]
> For consistency, I wonder if this could have been done as a method that can be passed to `Array.filter` like the `decideTrail` function is passed to `Array.map`. Composition is nice!

fyi we don't intend to keep this code. It was just a quick fix so once upstream filters to Ophan and Frontend are in place we can revert this PR.

**Comment by mchv** [comment]
> fyi we don't intend to keep this code. It was just a quick fix so once upstream filters to Ophan and Frontend are in place we can revert this PR.

@arelra 👍 and thank you for fixing this Yesterday 💎!

I think we have talked in the past about that feature being implemented in the tools, so filtering not having to be done in the frontends (wed, apps, etc.). @davidfurey @twrichards @CDicksonG Does that ring any bell? 

Regarding the implementation, I think we have already added some filtering logic (sensitive content) in `MostPopular` when removing sensitive content (https://github.com/guardian/frontend/pull/25552/file)

If not in configurable in `Composer`, it will probably make sense in the future to have filtering logic in `Content API` to avoid duplication in `MAPI` as well (see https://github.com/guardian/mobile-apps-api/pull/2818 from @waisingyiu)

**Comment by arelra** [comment]
To follow up this change has been reverted by:
- #9548

as filtering of most read has been moved upstream to Ophan by:

- https://github.com/guardian/ophan/pull/5661
- https://github.com/guardian/ophan/pull/5663

So this should apply to all downstream clients i.e. Frontend, CAPI (via [shuttlerun](https://github.com/guardian/content-api/blob/c3bf978412b642889295e925e2b450352af792b0/shuttlerun/README.md?plain=1)) and MAPI cc @waisingyiu

**Pull request merged**